docs/source/services/timelock_service/manual-migration.rst

```diff
@@ -0,0 +1,121 @@
+.. _manual-timelock-migration:
+
+Manual Migration to Timelock Services
+=====================================
+
+.. warning::
+
+   This is not a recommended way to migrate to external timelock server. We have **automatic** migrations in place.
```

Comment:
I'd recommend adding the caveat that this is not recommended _if you're using Cassandra KVS_ (you can't do automigration if not on cassandra)

Also, nit: `server` vs `services` in the title.